libs/platform/src/lib/components/form/combobox/combobox/combobox.component.scss

```diff
@@ -25,3 +25,12 @@
         }
     }
 }
+
+.fd-list__group-header.ng-star-inserted{
```

Comment:
You can't have selector `.ng-star-inserted`. It's Angular specific and if we add it in Fundamental-styles it won't work in the other libs (fundamental-react and fundamental-vue)